test/e2e/s3/cross-account.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+ROOT_DIR="$THIS_DIR/../../.."
+SCRIPTS_DIR="$ROOT_DIR/scripts"
+
+source "$SCRIPTS_DIR/lib/common.sh"
+source "$SCRIPTS_DIR/lib/k8s.sh"
+source "$SCRIPTS_DIR/lib/testutil.sh"
+
+
+# AWS_ACCOUNT_ID_ALT should be configured to allow cross account resources management
+# from AWS_ACCOUNT_ID
+AWS_ACCOUNT_ID_ALT=${AWS_ACCOUNT_ID_ALT:-""}
+AWS_PROFILE_ALT=${AWS_PROFILE_ALT:-""}
```

Comment:
please work with @smuralee to get the self-hosted runners set up with an appropriate alternate profile/account ID for the e2e tests that will run on new PRs. :)

Reply:
Sure! Will reach out ASAP.